### DISCUSSION TITLE
curation for GrantRound

### DISCUSSION THREAD

**Discussion by thelostone-mc** [dcgtc/dgrants#323] [Architecture & Design]
#### Description

Right now grants have no curation / means of grouping them
When a grant round is created , we have to 

- manually add/edit the grants in the round
- upload to IPFS 

This does not scale.
Before GR12 -> do we focus of doing at-least some means of tagging/curation to be able to group grants to make it easier to run rounds.

**Example:**
- Create Round for grants which are from Uniswap
- Create Round for grants created from south Africa
- Create Round for grants which are dApp 

AKA there are multiple params which can be used to group grants
How do we design for this ?

**Comment by phutchins** [comment]
I propose that we call this filtering instead of curation. I think of curation as what is displayed or allowed to be viewed in the explorer, and then filtering is taking that list and creating sub sets based on the criteria.
